saleor/graphql/account/mutations/base.py

```diff
@@ -129,7 +129,7 @@ def perform_mutation(cls, _root, info, **data):
             )
 
         try:
-            user = models.User.objects.get(email=email)
+            user = models.User.objects.get(email=email, is_active=true)
```

Comment:
`True`

Reply:
Hello @korycins, raised point is fixed. Regards.